### ISSUE TITLE
Search function not returning results.

### ISSUE THREAD

**Issue by betatester3016** [divingmule/plugin.audio.iheart#10]
OS: CoreELEC 19.3 (Matrix)

Using the search function returns an error. 

[Kodi Error Log](http://ix.io/3LFD)
```

2022-01-09 13:31:44.339 T:5600     INFO <general>: [plugin.audio.iheart-2.0.1]: Addon args: {'id': 'None', 'mode': 'menu', 'name': 'Menu', 'url': 'Menu'}
2022-01-09 13:32:44.177 T:5600    ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                                    - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                                   Error Type: <class 'KeyError'>
                                                   Error Contents: 'results'
                                                   Traceback (most recent call last):
                                                     File "/storage/.kodi/addons/plugin.audio.iheart/default.py", line 426, in <module>
                                                       display_menu()
                                                     File "/storage/.kodi/addons/plugin.audio.iheart/default.py", line 281, in display_menu
                                                       search()
                                                     File "/storage/.kodi/addons/plugin.audio.iheart/default.py", line 321, in search
                                                       if 'podcasts' in data['results'] and data['results']['podcasts']:
                                                   KeyError: 'results'
                                                   -->End of Python script error report<--

```
**Screenshots**

![screenshot00000](https://user-images.githubusercontent.com/49594656/148698119-32436901-af78-4efb-9918-906099dd7ebd.png)

![screenshot00001](https://user-images.githubusercontent.com/49594656/148698134-36727e84-0e0b-4a88-a888-9b58568b48f1.png)

![screenshot00002](https://user-images.githubusercontent.com/49594656/148698130-9401ef4d-bd2c-42f1-8196-e7ea12215089.png)

![screenshot00003](https://user-images.githubusercontent.com/49594656/148698135-7e9a73aa-549b-45ec-9001-bf3aad4f68e5.png)

**Comment by divingmule** [comment]
Thanks for bringing this to my attention and sorry for it taking so long for me to get to. 

I've released v2.0.2 for Matrix with a fix. Check the releases tab.

**Comment by betatester3016** [comment]
Thanks for fixing the search function.
